proto/ibc/applications/fee/v1/query.proto

```diff
@@ -26,7 +26,7 @@ service Query {
         "{packet_id.sequence}/incentivized_packet";
   }
 
-  // Gets all incentivized packets for a specific channel
+  // IncentivizedPacketsForChannel retrieves all incentivized packets for a specific channel
```

Comment:
Could you run `make proto-all` and `make proto-swagger-gen` please? :)

Reply:
I don't quite understand

Reply:
from the main `ibc-go` repo on your local filesystem invoke `make proto-all && make proto-swagger-gen`. Alternatively, if you allow maintainers to push to your branch, this can be done by someone from the team.